src/main/resources/application.yml

```
@@ -39,3 +39,17 @@ jwt:
     secretKey: umceightfightingjwttokenauthentication
     expiration:
       access: 14400000 # 4시간
+
+cloud:
+  aws:
+    s3:
+      bucket: umc-8th-dev
+      path:
+        review : review
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)